bin/rspecq

```
@@ -72,6 +72,11 @@ OptionParser.new do |o|
     opts[:max_requeues] = v
   end
 
+  o.on("--fail-fast N", Integer, "Stop running the suite after N failed tests" \
```

Can we support fail-fast without a N which would default to 1 (this would mimic rspec).

---

I will see what I can do

---

a1f7fda

---

How about:

> Exit with a non-zero status after N failed examples.

I believe we should default this to `nil` (don't fast fail).

---

We do. If you omit, the flag it wont fail fast. If you just add it, it will be 1 (Default value) and if you explicitly set it, it will have that number